platform/ddf-theme/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "ddf-theme",
+  "version": "0.0.1",
+  "description": "DDF's Bootstrap Theme",
+  "main": "index.js",
+  "scripts": {
+    "copy-fonts": "cp -R node_modules/bootswatch/fonts/ src/main/resources/webapp/fonts/bootstrap && cp -R src/main/webapp/google-fonts/ src/main/resources/webapp/fonts",
```

Comment:
❓ will these commands work on windows?

Reply:
good catch. I'm fairly certain the `cp` equivalent on windows is `copy`. We may need to come up with a windows specific script and kick it off if windows is detected.

Reply:
I have a solution for making it windows compatible, I could use npm modules rimraf, mkdirp, and cpr.